src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/services/TaskManagementServiceTest.java

```diff
@@ -2606,287 +1525,6 @@ void should_throw_task_not_found_exception_when_task_resource_not_found_and_feat
         }
     }
 
-    @Nested
-    @DisplayName("searchWithCriteria()")
```

Comment:
@DisplayName("searchWithCriteria()") in task management service is completed removed as it calls camunda api, instead now its cftqueryservice searchwithcriteria, already tests covered for thats in CFTqueryServiceTests

Since these method is completely removed from TaskManagementService, corresponding tests were also removed
so for R2 calls are only made from CFTQueryService not from TaskManagementService